docs/content/preview/tutorials/azure/azure-app-service.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Build Applications Using Azure App Service and YugabyteDB
+headerTitle: Use Azure App Service with YugabyteDB
+linkTitle: Azure App Service
+description: Build Applications Using Azure App Service and YugabyteDB
+headcontent: Extend scalability and reliability to the data layer
+image: /images/tutorials/azure/icons/App-Service-Icon.svg
+menu:
+  preview:
+    identifier: tutorials-azure-app-service
+    parent: tutorials-azure
+    weight: 10
+type: docs
+---
+
+In this tutorial we'll walk you through the steps to build and deploy a scalable web application using YugabyteDB and Azure App Service.
+
+Deploying to Azure App Service allows developers to leverage cloud infrastructure and build more scalable and available applications. Connecting these applications to YugabyteDB extends this scalability and reliability into the data layer!
+
+In this tutorial you will:
+
+1. Examine YB Shoes, a sample eCommerce dashboard application.
+1. Create a web application on Azure App Service.
+1. Provision a database cluster on YugabyteDB Managed and seed it with data.
+1. Deploy YB Shoes to Azure App Service from a local repository.
+
+Let's start by introducing YB Shoes and its functionality.
+
+## Introducing YB Shoes
+
+YB Shoes is an eCommerce dashboard that displays current shoe inventory and recent sales. YB Shoes is backed by a Node.js server which connects to YugabyteDB Managed. The queried inventory and sales data is displayed on a ReactJS frontend.
+
+First, visit GitHub for the [web application](https://github.com/YugabyteDB-Samples/yugabytedb-azure-app-service-demo-nodejs) you will be deploying to Azure App Service.
```

Comment:
"visit GitHub for the web application" - From this description, I'm not sure if I should download the app or do perhaps something else. In the event, we're just reviewing the app/code, right?